azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/inbox_manager.py

```diff
@@ -77,10 +77,11 @@ def get_method_request_inbox(self, method_name=None):
         return inbox
 
     def clear_all_method_requests(self):
```

Comment:
>clear_all_method_requests [](start = 8, length = 25)

nit: just to make this clear i'd suggest renaming this to "delete_method_inboxes". deleting inboxes implies clearing the method requests, the other way doesn't seem true. not a blocker.